patches/chrome-browser-resources-settings_shared-controls-controlled_radio_button.html.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/resources/settings_shared/controls/controlled_radio_button.html b/chrome/browser/resources/settings_shared/controls/controlled_radio_button.html
+index b2c2c288a5fe0940f99843a4f556cfb4d73ef5bd..e9095c6c81921cd861cd5a6b1b702ee8ba5845ee 100644
+--- a/chrome/browser/resources/settings_shared/controls/controlled_radio_button.html
++++ b/chrome/browser/resources/settings_shared/controls/controlled_radio_button.html
+@@ -1,6 +1,6 @@
+ <style
+ <if expr="not chromeos_ash">
+-  include="cr-radio-button-style"
```

Comment:
@emerick didn't we have something similar in cr114? What did we end up doing and decided was the right fix? cc @petemill

Reply:
Not really. I was possibly just trying to fix conflict resolution.

Reply:
Reverted here 85cafc7096c9bda3dd6fdf0b713d39e7e0b74605.